tests/RealMeServiceTest.php

```diff
@@ -77,8 +77,12 @@ public function testGetAuth()
 
         // Identity Provider settings
         $idpData = $auth->getSettings()->getIdPData();
-        $this->assertSame('https://mts.realme.govt.nz/saml2', $idpData['entityId']);
-        $this->assertSame('https://mts.realme.govt.nz/logon-mts/mtsEntryPoint', $idpData['singleSignOnService']['url']);
+        $expected = 'https://mts.login.realme.govt.nz' .
```

Comment:
Do we know why the expected URLs change all the sudden? Are we doing a live HTTP request to fetch them?

Reply:
See linked [PR](https://github.com/silverstripe/silverstripe-realme/pull/66)